build.gradle

```diff
@@ -2,7 +2,9 @@
 apply plugin: 'maven-publish'
 
 group = 'org.elasticsearch'
-version = '4.4.0'
+String upstreamVersion = '4.4.0'
+String localPrerelease = '1'
+version = "${upstreamVersion}-${localPrerelease}"
```

Comment:
Sorry, can we please separate the version extension (if we decide to do it) from the aarch64 addition?